Does not compile on rustc 1.14.0-nightly (3f4408347 2016-10-27)

```
error[E0063]: missing field `is_shorthand` in initializer of `syntax::ast::Field`
    --> src/expr.rs:1120:21
     |
1120 |         let field = ast::Field {
     |                     ^^^^^^^^^^ missing `is_shorthand`
```

`is_shorthand` added in https://github.com/rust-lang/rust/pull/11994

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -12,7 +12,7 @@ with-syntex = ["syntex_syntax"]
 unstable-testing = ["clippy", "compiletest_rs"]
 
 [dependencies]
-syntex_syntax = { version = "^0.46.0", optional = true }
+syntex_syntax = { version = "^0.47.0", optional = true }
 clippy = { version = "0.*", optional = true }
 compiletest_rs = { version = "^0.2.0", optional = true }
 

--- a/src/expr.rs
+++ b/src/expr.rs
@@ -1121,6 +1121,7 @@ impl<I, F> Invoke<P<ast::Expr>> for ExprStructFieldBuilder<I, F>
             ident: respan(self.builder.span, self.id.to_ident()),
             expr: expr,
             span: self.builder.span,
+            is_shorthand: false,
         };
         self.builder.fields.push(field);
         self.builder

